packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.ts

```diff
@@ -1,16 +1,14 @@
-import { style } from '../styles';
```

Comment:
Removes the `styles.ts` file since it's no longer being used and also refactored the stylesheets wherever possible either by drying them up or removing `!important` syntax